.github/scripts/rust/setup.py

```diff
@@ -14,6 +14,9 @@
 # Exclude the stable channel for these crates
 DISABLE_STABLE_PATTERNS = ["packages/engine**"]
 
+# Exclude these crates to run `rustdoc` at
+DISABLE_DOC_PATTERNS = ["packages/engine"]
```

Comment:
This could be a little misleading. It's just not running the docs for the binary right? I'm assuming we still run the docs for all of the sub crates? If so then can we add a little comment explaining it